man/org.freedesktop.systemd1.xml

```diff
@@ -1338,7 +1342,9 @@ node /org/freedesktop/systemd1 {
       string guaranteed, and new fields may be added any time, and old fields removed. The general structure
       may be rearranged drastically between releases. This is exposed by
       <citerefentry><refentrytitle>systemd-analyze</refentrytitle><manvolnum>1</manvolnum></citerefentry>'s
-      <command>dump</command> command. The <function>DumpByFileDescriptor()</function> method is identical to
+      <command>dump</command> command. Similarly, <function>DumpPatterns()</function> allows to return the
```

Comment:
"allows to return" → "returns"

"a limited number of units whose names, possibly including glob expressions, are specified as arguments" → "units whose names match the glob expressions specified in the `<varname>patterns</varname>` argument".
(With `"*"` it's not "limited".)